java/registry/src/main/java/dev/sunbirdrc/registry/helper/RegistryHelper.java

```diff
@@ -16,6 +16,7 @@
 import dev.sunbirdrc.pojos.attestation.Action;
 import dev.sunbirdrc.pojos.attestation.States;
 import dev.sunbirdrc.pojos.attestation.exception.PolicyNotFoundException;
+import dev.sunbirdrc.registry.Constants;
```

Comment:
is this import used?

Reply:
Nope, I have removed might have used it while Writing a separate function in the helper for revoke. But then had to make changes in the flow. So might have missed removing the import. Have removed it now